xbmc/pvr/addons/PVRClients.cpp

```diff
@@ -176,6 +178,7 @@ void CPVRClients::Unload(void)
   m_strPlayingClientName = "";
 
   m_clientMap.clear();
+  m_loadedEvent.Reset();
```

Comment:
from the looks of it it doesn't, which means that we can block up to 5 seconds now on exit or when restarting the pvr manager, not good